src/java/org/apache/cassandra/index/accord/RoutesSearcher.java

```diff
@@ -97,22 +98,80 @@ public Entry next()
         }
     }
 
-    public void intersects(int store, TokenRange range, TxnId minTxnId, Timestamp maxTxnId, Consumer<TxnId> forEach)
+    private CloseableIterator<Entry> searchKey(int store, AccordRoutingKey key)
     {
-        intersects(store, range.start(), range.end(), minTxnId, maxTxnId, forEach);
+        RowFilter rowFilter = RowFilter.create(false);
+        rowFilter.add(participants, Operator.GTE, OrderedRouteSerializer.serializeRoutingKey(key));
+        rowFilter.add(participants, Operator.LTE, OrderedRouteSerializer.serializeRoutingKey(key));
+        rowFilter.add(store_id, Operator.EQ, Int32Type.instance.decompose(store));
+
+        var cmd = PartitionRangeReadCommand.create(cfs.metadata(),
+                                                   FBUtilities.nowInSeconds(),
+                                                   columnFilter,
+                                                   rowFilter,
+                                                   limits,
+                                                   dataRange);
+        Index.Searcher s = index.searcherFor(cmd);
+        try (var controller = cmd.executionController())
```

Comment:
```suggestion
        try (ReadExecutionController controller = cmd.executionController())
```

since we want to ban `var`